src/book/02-system/04-other/01-nubook.mdx

```diff
@@ -87,7 +85,7 @@ You'll need [Git](https://git-scm.com/), [Node.js](https://nodejs.org/en/) and [
 All additions and edits are done through GitHub pull requests. To add or edit content:
 
 1. Clone the repo, install dependencies, and run the development server as shown in [Getting started](#getting-started) above.
-2. Create a new branch for your changes, in the format `your_last_name/short_description_of_change`:
+2. Create a new branch for your changes, in the format `surname/short_description_of_change`:
```

Comment:
I think `your_surname` should be kept as someone new to this might be wondering "what surname?"

Reply:
This was more to keep consistency with the Git guide page. We should change that one too if we're changing this one.

Reply:
which I see you have suggested on the Git guide PR